src/index.htm

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Calculatron 9000</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Nunito"
+    />
+    <link rel="icon" type="image/x-icon" href="/public/favicon.ico" />
+    <script src="/scripts/calculator.js" defer></script>
+  </head>
+</html>
+<body>
+  <main class="calculatorGrid">
+    <div class="display">
+      <h3 data-previous-operand class="previousOperand"></h3>
+      <h1 data-current-operand class="currentOperand"></h1>
+      </h1>
+    </div>
+    <button data-number="7">7</button>

```

Comment:
Might be an idea to not duplicate these data.
E.g. could just do `<button data-calc-role="number-input">5</button>`.
If the numbers are only being stored as strings on the DOM anyway, then it could just as well be done as innerText.
An alternative, would be to dynamically generate the 0..9 buttons (via JavaScript) on page load.